internal/controller/vrg_volrep.go

```diff
@@ -114,6 +127,8 @@ func (v *VRGInstance) reconcileVolRepsAsPrimary() {
 
 		log.Info("Successfully processed VolumeReplication for PersistentVolumeClaim")
 	}
+
+	v.reconcileVolGroupRepsAsPrimary(groupPVCs)
```

Comment:
If a PVC in the above loop was skipped or requeued we would still protect it here with a VGR as the PVC already is labbeled with the required CG label. This needs some attention to address when a group VGR would be created.